az-digital/az_quickstart
# Closes #1530 Adding theme-debug tooling for lando and ddev.

## Description

Adding tooling to easily enable theme debugging.

## Related issues
Closes #1530 
Decided on not including the enable devel module step, since that is easy enough to do with
`lando drush en -y devel`

## How to test
This is a developer convenience command when using lando to update theme files.

```
git clone https://github.com/az-digital/az_quickstart.git issues-1530
cd issues-1530
git checkout issues/1530
lando start
lando install
lando theme-debug

```
This is a developer convenience command when using ddev to update theme files.

```
git clone https://github.com/az-digital/az_quickstart.git issues-1530
cd issues-1530
git checkout issues/1530
ddev config --project-type php
ddev start
ddev install
ddev theme-debug

```

Theme debugging should successfully enable.


## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
- Minor release changes

### Drupal core
- Patch release changes
- Minor release changes

### Drupal contrib projects
- Patch release changes
- Minor release changes

## Checklist
